CHANGES.rst

```diff
@@ -7,8 +7,10 @@ since the branch point at Zope 4.1.2.
 The change log for the previous version, Zope 4, is at
 https://github.com/zopefoundation/Zope/blob/4.x/CHANGES.rst
 
-5.14 (unreleased)
------------------
+6.0 (unreleased)
```

Comment:
Should we actually call it version 6.0 or would 5.14 be enough.

For a version 6 we should also apply for a new trove classifier, I think.

Reply:
I think that depends on what we do about horse-with-no-namespace.

If you add that as a dependency then a Zope upgrade won't break any deployment and it would be OK to call it 5.14.

If you don't add it as a dependency and choose to e.g. just document it as a band-aid if something breaks then it should be 6.0 because it will break existing deployments without additional steps like adding horse-with-no-namespace.